### DISCUSSION TITLE
希望进程管理器支持列表显示进程的文件名称(如jar包), 当前端口信息

### DISCUSSION THREAD

**Discussion by NilServer404** [1Panel-dev/1Panel#6481] [Ideas]
### 1Panel 版本

v1.10.13-lts

### 请描述您的需求或者改进建议

[FEATURE]希望进程管理器支持列表显示进程的文件名称(如jar包), 当前端口信息

**Comment by zhengkunwang223** [comment]
![image](https://github.com/user-attachments/assets/25e44a2d-341b-41fb-b98a-07dab357c353)
你是要把详情中的显示到列表中吗？
有点多 不好显示

**Comment by NilServer404** [comment]
不用  就增加二进制文件名和当前使用端口就可以  谢谢您

**Comment by zhengkunwang223** [comment]
> 不用  就增加二进制文件名和当前使用端口就可以  谢谢您

如何分辨是你需要的文件和端口呢？

**Comment by AnkioTomas** [comment]
端口应该只要Listening状态的
至于文件，我觉得他可能需要类似于`ps -ef`这样的输出？

**Comment by toohandsome** [comment]
端口可以用 netstat 查吧，文件应该指的是 cmdline

**Comment by zhengkunwang223** [comment]
> 端口应该只要Listening状态的
> 
> 至于文件，我觉得他可能需要类似于`ps -ef`这样的输出？

如果要根据 pid 再次查询之后才能显示的数据放在列表显示会增加接口的读取时间
